backend/tests/courses/test_api.py

```diff
@@ -716,14 +716,14 @@ def test_patch(self):
         self.assertEqual(response.data["first_name"], "new_name")
         response = self.client.patch(
             "/accounts/me/",
-            json.dumps({"profile": {"phone": "3131234567"}}),
+            json.dumps({"profile": {"phone": "19178286431"}}),
```

Comment:
What's the reason for changing these numbers?

Reply:
They were invalid for some reason